setup.py

```diff
@@ -26,8 +26,8 @@
     if py_version < (3, 3) and not is_pypy: # PyPy3 masquerades as Python 3.2...
         raise RuntimeError('On Python 3, Pyramid requires Python 3.3 or better')
 else:
-    if py_version < (2, 6):
-        raise RuntimeError('On Python 2, Pyramid requires Python 2.6 or better')
+    if py_version < (2, 7):
+        raise RuntimeError('On Python 2, Pyramid requires Python 2.7 or better')
```

Comment:
I'm a little concerned about adding 2.6 as a hard fail considering pyramid does still run on 2.6 just fine. I'd prefer to wait to add a hard fail until (if) we start using 2.7-only features like set/dict comprehensions, etc. Even if the test suite stops working on 2.6 I don't see a reason to hard fail until the core library uses 2.7-only features.